src/pages/[id].page.jsx

```diff
@@ -29,20 +29,28 @@ export default function Details({ details }) {
   const [htmlCode, setHtmlCode] = useState('');
   const [cssCode, setCssCode] = useState('');
   const [javascriptCode, setJavascriptCode] = useState('');
+  const [canRenderSandpack, setCanRenderSandpack] = useState(false);
 
   // dynamic imports need to be asynchronous
   useEffect(() => {
-    const updateCodeBlocks = async () => {
-      const html = await import(`../code-examples/${slug}/html.js`);
-      setHtmlCode(html.default);
+    const updateSandpackExamples = async () => {
+      try {
+        const html = await import(`../code-examples/${slug}/html.js`);
+        setHtmlCode(html.default);
 
-      const css = await import(`../code-examples/${slug}/css.js`);
-      setCssCode(css.default);
+        const css = await import(`../code-examples/${slug}/css.js`);
+        setCssCode(css.default);
 
-      const javascript = await import(`../code-examples/${slug}/javascript.js`);
-      setJavascriptCode(javascript.default);
+        const javascript = await import(
+          `../code-examples/${slug}/javascript.js`
+        );
+        setJavascriptCode(javascript.default);
+        setCanRenderSandpack(true);
+      } catch (e) {
+        setCanRenderSandpack(false);
```

Comment:
We don't explicitly need to set this to `false` in the `catch` block (but it doesn't hurt anything to do so). I'd add a `console.log` that says something like `Couldn't find code examples for ${slug}.`

Reply:
Added the `console.log`: https://github.com/sparkbox/accessible-components/pull/132/commits/7412fb3461f583a6b0ffd2e548bfabdb865cb287